src/tools/rbd_mirror/image_map/RemoveRequest.cc

```diff
@@ -0,0 +1,133 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
```

Comment:
Combine ```RemoveRequest``` with ```UpdateRequest``` that can batch multiple remote/update operations w/ a single op to the OSD.